### ISSUE TITLE
mucon demo

### ISSUE THREAD

**Issue by nmcl** [jbosstm/conferences#3]
It doesn't appear to be written in a way that it can be built out of the box by someone who hasn't written it in the first place. For example in the README ...

```bash
mkdir demo; cd demo
mvn io.fabric8:vertx-maven-plugin:1.0.7:setup -DvertxVersion=3.4.2 -Ddependencies=web

But how does this relate to the contents of the directory?

total 2344
-rw-r--r--  1 marklittle  staff  1187657 22 Nov 15:35 Mucon2017TransactionalActors.pdf
-rw-r--r--  1 marklittle  staff     6367 22 Nov 15:35 README.md
drwxr-xr-x  6 marklittle  staff      204 22 Nov 15:35 common
drwxr-xr-x  4 marklittle  staff      136 22 Nov 15:35 demo_with_STM
drwxr-xr-x  4 marklittle  staff      136 22 Nov 15:35 demo_without_STM
drwxr-xr-x  4 marklittle  staff      136 22 Nov 15:35 flight
drwxr-xr-x  4 marklittle  staff      136 22 Nov 15:35 hotel
-rw-r--r--  1 marklittle  staff      963 22 Nov 15:35 pom.xml
drwxr-xr-x  4 marklittle  staff      136 22 Nov 15:35 stress
drwxr-xr-x  5 marklittle  staff      170 22 Nov 15:35 trip

And what about installing OpenShift? Are there pre-requisites assumed?

**Comment by mmusgrov** [comment]
@nmcl It was designed to be a live coding demo from zero to fully functional STM application (ie from an empty directory) and the instructions were reminders on how to do that. However, the demo_with_STM and demo_without_STM were backups so I can use those to indicate how to run the demo out of the box. I will update the README.

**Comment by nmcl** [comment]
OK but there could be a script that others follow to do the same live coding you did.

**Comment by mmusgrov** [comment]
fixed with commit 96e726f997995c3e3683bf3a4692f0443ab4059c
